### DISCUSSION TITLE
Unable to delete old template repositories now I am no longer a member of the organisation that owns them

### DISCUSSION THREAD

**Discussion by lhorrell99** [community/community#64659] [Repositories]
### Select Topic Area

Product Feedback

### Body

I've recently been in touch with Github support to ask about removing old repository templates that come up when I go to initialise a new repo. Since I am not the owner of these templates, and also no longer have access to the organisation that created them, I am unable to remove them. Not really a big issue, but I would love to see the ability to remove old template repos from inside my account settings added as a feature.

**Comment by BillRaymond** [comment]
This is a bigger issue for me because I create training videos and if I am testing templates, they show up -- and remain on -- the list. We need the ability to delete (or manage?) templates.
